common/dbus.hpp

```diff
@@ -21,6 +21,7 @@
 #include <memory>
 #include <string>
 #include <unordered_map>
+#include <utility>
 
 #ifdef MENDER_USE_ASIO_LIBDBUS
 #include <dbus/dbus.h>
```

Comment:
Nit: Pre-existing, but this is a `C` header, no?

Reply:
It's also a 3rd party library header so not entirely sure which rule applies here.

Reply:
Good point!

Reply:
Makes sense